_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,354 @@
+ Docker Hub if it exists locally which in our case it does. Do note that you 
+ still need to be logged on to Docker so that Kubectl can check if the image 
+ is up to date.
+ 
+We can check that a Pod is created for our application by running:
+
+```bash
+kubectl get pods
+```
+
+which should return a similar output to:
+
+```bash
+NAME                                       READY     STATUS    RESTARTS   AGE
+laravel-kubernetes-demo-7dbb9d6b48-q54wp   1/1       Running   0          18m
+```
+
+We can also use the Minikube GUI dashboard to monitor our cluster. The GUI 
+also helps with visualising most of the discussed concepts. To view the 
+dashboard, simply run the following:
+
+```bash
+minikube dashboard
+```
+
+or to acquire the dashboard's URL address:
+
+```bash
+minikube dashboard --url=true
+```
+
+which will return the IP address and the port number of the dashboard.
+
+__Exposing a service__
+
+So far we have created a deployment which is running our application's 
+container. With the container running, we need to expose the PHP's built-in web 
+server as a service.
+
+```bash
+kubectl expose deployment laravel-kubernetes-demo --type=NodePort --port=8181
+```
+
+and provided all went well, we should see a confirmation similar to:
+
+```bash
+service "laravel-kubernetes-demo" exposed
+```
+
+We can also view the running service under "Services" navigation menu within 
+the dashboard. A more exciting way to verify this deployment and the service 
+exposure is obviously viewing our running application in the browser 😊
+
+To obtain the URL of our application (service), we can use the following 
+command:
+
+```bash
+minikube service --url=true laravel-kubernetes-demo
+```
+
+which will output the IP address and port number similar to:
+
+```bash
+http://192.168.99.101:31399
+```
+
+or, launch the application directly in the browser:
+
+```bash
+minikube service laravel-kubernetes-demo
+```
+
+__Scaling__
+
+And that is it. We have successfully deployed our application in Kubernetes. 
+It's exciting, but what's the point of doing all of this? Well, we only have 
+one deployment with a single Pod running, provisioned to a Node with the 
+exposed web service. Let's replicate this service to two more Pods acting as 
+load balancers to our traffic.
+
+So that we understand where we are at this moment, run the following command 
+to get a list of desired and available Pods:
+
+```bash
+kubectl get deployment
+
+NAME                      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+laravel-kubernetes-demo   1         1         1            1           57m
+```
+
+The output will be "1" for each. We want to have 3 available Pods so let's 
+scale this up:
+
+```bash
+kubectl scale --replicas=3 deployment/laravel-kubernetes-demo
+deployment "laravel-kubernetes-demo" scaled
+```
+
+Done. We have replicated the initial Pod to another two, giving us three 
+Pods running this service. Running the `get deployment` command will verify 
```

Comment:
As above. Be careful with the word service. Here's a little bit clearer that service refers to the application.